templates/cli/lib/parser.js.twig

```diff
@@ -110,13 +114,45 @@ const drawJSON = (data) => {
 }
 
 const parseError = (err) => {
-    if(cliConfig.verbose) {
-        console.error(err);
-    }
+    if (cliConfig.report) {
+        (async () => {
+            let appwriteVersion = 'unknown';
+            const isCloud = globalConfig.getEndpoint().includes('cloud.appwrite.io') ? 'Yes' : 'No';
+
+            try {
+                const res = await fetch(`${globalConfig.getEndpoint()}/health/version`);
```

Comment:
why dont we use the sdkForProject or sdkForConsole to make this call ? 
Or even Client.call() ?

Reply:
Okay why not use the call method of the client class ?
https://github.com/appwrite/sdk-for-cli/blob/master/lib/client.js#L146

Reply:
I dont think its necessary to use a different pattern here to fetch data when we already have the call method

Reply:
Fixed